StreamChatUITestsAppUITests/Robots/UserRobot.swift

```diff
@@ -212,6 +212,81 @@ extension UserRobot {
         MessageListPage.list.press(forDuration: 0.1, thenDragTo: topMessage)
         return self
     }
+    
+    @discardableResult
+    func openComposerCommands() -> Self {
+        if MessageListPage.ComposerCommands.cells.count == 0 {
+            MessageListPage.Composer.commandButton.wait().tap()
+        }
+        return self
+    }
+    
+    @discardableResult
+    func sendGiphy(useComposerCommand: Bool = false, shuffle: Bool = false) -> Self {
+        let giphyText = "Test"
+        if useComposerCommand {
+            openComposerCommands()
+            MessageListPage.ComposerCommands.giphyImage.wait().tap()
+            sendMessage("\(giphyText)")
```

Comment:
shouldn't it be `/giphy \(giphyText)`?

Reply:
Not really. In this case, we're tapping on the `Giphy` command and then just typing the text. The whole condition looks like this:
```swift
        if useComposerCommand {
            openComposerCommands()
            MessageListPage.ComposerCommands.giphyImage.wait().tap()
            sendMessage("\(giphyText)")
        } else {
            sendMessage("/giphy\(giphyText)")
        }
```

Reply:
Ah, ok. When the command is selected, it's just the arguments that should be in the message text input view. All good then 👍

Reply:
Don't we need a space between command and its' args? `sendMessage("/giphy\(giphyText)")`

Reply:
> Don't we need a space between command and its' args? `sendMessage("/giphy\(giphyText)")`

As soon as the user types `/giphy` we clear the text field. The user will type text without adding a space. At least it's the current behavior.